library/globals.inc.php

```diff
@@ -1866,7 +1866,12 @@
       '0',
       xl('Enable Onsite Patient Portal.')
     ),
-
+    'portal_statements' => array(
```

Comment:
I would then remove this global since it is unrelated to the eob totals feature.